internal/backends/collection.go

```diff
@@ -190,6 +192,30 @@ func (cc *collectionContract) Explain(ctx context.Context, params *ExplainParams
 	return res, err
 }
 
+// CollectionStatsParams represents the parameters of Collection.Stats method.
+// The prefix Collection is used to prevent name crash with Database.Stats parameter.
+type CollectionStatsParams struct{}
```

Comment:
Ah, now we have a clash… Let's rename `StatsParams/Result` to `DatabaseStatsParams/Result` then too

Reply:
Let's do that for the consistency. Updated.